### PR TITLE
feat: blog post hero

### PR DESCRIPTION
**Pull request by feedm3**
This PR contains some blog and blog post adjustments:

- Blog post blockquote gets now formatted correctly
- Blog post headers now have the correct tags
<details>
 <summary>Blog post can now have a hero image (background of header needs to be discussed with designers)</summary>
Screenshot 2020-08-03 at 14 26 46@2x
</details>
<details>
 <summary>Blog overview has correct margins</summary>
Screenshot 2020-08-03 at 14 26 30@2x
</details>

### CODE REVIEW

File: src/components/image/cloudinary-image.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+const StyledImage = styled.img`
+  width: 100%;
+`;
+
+interface CloudinaryImageProps {
+  src: string;
+  alt: string;
+}
+
+export const ResponsiveCloudinaryImage: React.FC<CloudinaryImageProps> = (
```
Comment (georgiee) [inline src/components/image/cloudinary-image.tsx:21]: Finally a separate file for that powerful component

File: src/components/image/cloudinary-image.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    250,
+    [windowWidth],
+  );
+
+  return <StyledImage src={imageSrc} alt={props.alt} ref={imageRef} />;
+};
+
+const BackgroundCloudinaryImage = styled.div<{ src: string }>`
+  height: 344px;
+  object-fit: scale-down;
```
Comment (georgiee) [inline src/components/image/cloudinary-image.tsx:51]: wanted to ask if this still needs a polyfill. searched for myself instead. we are good!

File: src/components/image/cloudinary-image.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  const [imageWidth, setImageWidth] = useState(0);
+  const [imageSrc, setImageSrc] = useState('');
+  const { width: windowWidth } = useWindowSize();
+
+  useDebounce(
+    () => {
+      const width = imageRef.current?.clientWidth || 0;
+      const newImageWidth = width <= IMAGE_MAX_WIDTH ? width : IMAGE_MAX_WIDTH;
+
+      // we only change the width if the picture gets better/larger
```
Comment (georgiee) [inline src/components/image/cloudinary-image.tsx:75]: nice, makes sense. the machine doesn't get weaker just by making the window smaller so this is a nice idea. I would just have blindly followed the logic backwards usually.

File: src/components/util/cloudinary-util.test.ts [hunk 4]
```diff
@@ -1,20 +1,24 @@
...
+    ).not.toThrow();
   });
 
   it('should detect an incorrect url', () => {
-    expect(isCloudinaryUrl(undefined as any)).toBeFalsy();
-    expect(isCloudinaryUrl(null as any)).toBeFalsy();
-    expect(isCloudinaryUrl('')).toBeFalsy();
-    expect(isCloudinaryUrl('cloudinary.com/satellytes')).toBeFalsy();
-    expect(isCloudinaryUrl('test.com')).toBeFalsy();
+    expect(() => assertCloudinaryUrl(undefined as any)).toThrow();
```
Comment (georgiee) [inline src/components/util/cloudinary-util.test.ts:17]: I love those inline actions feedback 😳

**Review by georgiee — APPROVED** [review]
Thank you. I provided some inline comments 
but nothing to take actions. So from my high-level review ready to go 👍

File: src/components/util/cloudinary-util.test.ts (hunk 4 above)
Comment (feedm3) [reply src/components/util/cloudinary-util.test.ts:17]: thanks 🙂  got the idea from google guava library (utility library for java)

**Pull request merged**